backend/tracim_backend/lib/utils/request.py

```diff
@@ -111,7 +111,7 @@ def candidate_workspace(self) -> Workspace:
         )
 
     @property
-    def candidate_content_type(self) -> ContentType:
+    def candidate_content_type(self) -> TracimContentType:
```

Comment:
what is a candidate_content_type? The docstring is not clear about what this does.

Reply:
not application related, will be fixed in #2626